Gateway/Request/Header/ExternalPlatformHeaderDataBuilderInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Adyen\Payment\Gateway\Request\Header;
+
+interface ExternalPlatformHeaderDataBuilderInterface
+{
+    const EXTERNAL_PLATFORM_NAME = 'external-platform-name';
+    const EXTERNAL_PLATFORM_VERSION = 'external-platform-version';
+    const EXTERNAL_PLATFORM_EDITION = 'external-platform-edition';
+    const EXTERNAL_PLATFORM_FRONTEND_TYPE = 'external-platform-frontendtype';
+
+    const FRONTEND_TYPE = 'frontendType';
+    const FRONTEND_TYPE_HEADLESS = 'headless';
```

Comment:
these constants have nothing to do with headers, is this the best place for them to live?

Reply:
const FRONTEND_TYPE = 'frontendType' is part of the header, it goes as the value for $header[EXTERNAL_PLATFORM_FRONTEND_TYPE]. About the other one, it was already so I kept it to keep all frontend related constants in one place, and since it not being used anywhere I am not sure where does it belong anyway.